baseline-network-linux-hld.md

```diff
@@ -0,0 +1,163 @@
+# Unified Open Networking Hardware Support
+
+[[_TOC_]]
+
+## Scope
+
+The current open network operating system (NOS) market is fragmented and has
```

Comment:
Should we call out what PAS/NOSes are included in this list? Possibly just the fully open source ones ex: community SONiC as SONiC has many bespoke Enterprise versions from different vendors.